backend/manager/modules/vdsbroker/src/main/java/org/ovirt/engine/core/vdsbroker/jsonrpc/JsonRpcVdsServer.java

```diff
@@ -1433,6 +1433,7 @@ public StatusOnlyReturn diskReplicateStart(String vmUUID, Map srcDisk, Map dstDi
                 new RequestBuilder("VM.diskReplicateStart").withParameter("vmID", vmUUID)
                         .withParameter("srcDisk", srcDisk)
                         .withParameter("dstDisk", dstDisk)
+                        .withParameter("needExtend", false)
```

Comment:
You cannot add this by default, the caller should set it. If this is only for testing, add a comment that this is temporary hack for testing.

Reply:
yeah, something is fishy here

Reply:
I think this is the WIP part, code that works only for testing certain case before the real implementation is added.